Frontend/VIAProMa/Assets/Scripts/Analytics/FileExport/DropdownCapability.cs

```diff
@@ -0,0 +1,116 @@
+using i5.VIAProMa.DataModel.API;
+using i5.VIAProMa.DataModel.ReqBaz;
+using i5.VIAProMa.UI;
+using i5.VIAProMa.UI.DropdownMenu;
+using i5.VIAProMa.UI.ListView.Core;
+using i5.VIAProMa.UI.ListView.Strings;
+using i5.VIAProMa.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace i5.VIAProMa.Shelves.IssueShelf
+{
+    public class DropdownCapability : MonoBehaviour, IWindow
+    {
+        [Header("References")]
+        [Header("UI Elements")]
+        [SerializeField] private StringDropdownMenu sourceSelection;
+        [SerializeField] private StringListView listView;
+
+        private Project[] projects;
+        private Category[] categories;
+
+        public event EventHandler SourceChanged;
+        public event EventHandler WindowOpened;
+        public event EventHandler WindowClosed;
+
+        private bool isConfiguring = true;
+
+        public ExportSelection ChosenExportSelection;
+
+        public IDropdownConfiguration DropdownConfiguration { get; private set; }
+
+        public bool WindowEnabled
+        { // not needed for configuration window => does not have an effect
+            get; set;
+        }
+        public bool WindowOpen { get; private set; } = true;
+
+        public bool ExternallyInitialized { get; set; } = false;
+
+        private void Awake()
+        {
+            
+            if (sourceSelection == null)
+            {
+                SpecialDebugMessages.LogMissingReferenceError(this, nameof(sourceSelection));
+            }
+            sourceSelection.ItemSelected += SetExportSelection; 
+        }
+
+        private void Start()
+        {
+            if (!ExternallyInitialized)
+            {
+                Initialize();
+            }
+        }
+
+        public void Initialize()
+        {
+            // populate the source dropdown menu with the available data sources
```

Comment:
Please stick to the C# style guide and capitalize the first letter of a comment. Also add a dot afterwards